qweb_usertime/__openerp__.py

```diff
@@ -1,28 +1,10 @@
 # -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
 #    This module copyright (C) 2015 Savoir-faire Linux
 #    (<http://www.savoirfairelinux.com>).
```

Comment:
@GillesTephaneMeyomesse Use the template header in https://github.com/OCA/maintainer-tools/tree/master/template/module